doc/changelog.md

```diff
@@ -1,151 +1,135 @@
-# Release 0.2.2-dev
+# Release 0.3.0
 
 <h3>New features</h3>
 
-* Catalyst users can now use Python control flow statements in their programs without having to
-  explicitly use the functions `cond`, `for_loop`, and `while_loop`!
-  [#235](https://github.com/PennyLaneAI/catalyst/pull/235)
+* Catalyst now builds on macOS, with macOS binary wheels available. For more details
+  on the changes involved to support macOS, please see the improvements section.
+  [(#229)](https://github.com/PennyLaneAI/catalyst/pull/230)
+  [(#232)](https://github.com/PennyLaneAI/catalyst/pull/232)
+  [(#233)](https://github.com/PennyLaneAI/catalyst/pull/233)
+  [(#234)](https://github.com/PennyLaneAI/catalyst/pull/234)
 
-  A new experimental feature was added that automatically converts Python control flow statements
-  like `if`, `for`, and `while` into their equivalent functional forms provided by Catalyst.
-  The feature is based on the AutoGraph module from TensorFlow, and requires a working TensorFlow
-  installation.
+* Write Catalyst-compatible programs with native Python conditional statements.
+  [(#235)](https://github.com/PennyLaneAI/catalyst/pull/235)
 
-  For example, the following can now be expressed much more succinctly using Python control flow:
+  Autograph is a new, experimental, feature that automatically converts
+  Python conditional statements like `if`, `else`, and `elif`,
+  into their equivalent functional forms provided by Catalyst (such as
+  `catalyst.cond`).
+
+  This feature is currently opt-in and experimental, and requires setting the
+  `autograph=True` flag in the `qjit` decorator:
 
   ```python
+  dev = qml.device("lightning.qubit", wires=1)
+
   @qjit(autograph=True)
-  @qml.qnode(qml.device("lightning.qubit", wires=n))
+  @qml.qnode(dev)
   def f(x):
-
-    for i in range(n):
-      qml.Hadamard(i)
-
       if x < 0.5:
-        y = jnp.sin(x)
+          qml.RY(jnp.sin(x), wires=0)
       else:
-        y = jnp.cos(x)
-
-      qml.RY(y, wires=i)
+          qml.RX(jnp.cos(x), wires=0)
 
-    return qml.probs()
+      return qml.expval(qml.PauliZ(0))
   ```
 
-  , which previously would be expressed as:
+  This feature is based on the AutoGraph module from TensorFlow, and
+  requires a working TensorFlow installation be available. In addition,
+  Python loops (`for` and `while`) are not currently supported, and do
+  not work in Autograph mode.

```

Comment:
@dime10 you are probably already aware, but I found out that `@qjit(autograph=True)` will fail if the contained function has non-recognized control flow (e.g., `for`, `while`), rather than gracefully just fallback to python interpreter for those.

Which is fine, the autograph feature isn't complete yet!

Reply:
Yeah good point, another reason I didn't want to release it yet 😅